### PR TITLE
updating 'webhook' to 'postee'

### PR DESCRIPTION
**Pull request by cpt-redbeard**
updating references to postee

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,25 +40,25 @@ For more information see "Ignore-Registry" and "Ignore-Image-Name" tokens in the
...
-1. Clone this project: 
+1. Clone this project:
 ```bash
 git clone git@github.com:aquasecurity/postee.git
 ```
 
-2. Build the postee Docker image: 
+2. Build the postee Docker image:
 ```bash
 docker build -t postee:latest .
```
Comment (jerbia) [inline README.md:53]: Can you change the image name to aqua-postee? I see that in the run command you aree using aqua-postee.

**Review by jerbia — CHANGES_REQUESTED** [review]
Can you change the image name in the documentation to "aqua-postee"? I see places where it says "postee"

**Comment by cpt-redbeard** [comment]
REDME updated with naming convention change.

**Review by jerbia — CHANGES_REQUESTED** [review]
@cpt-redbeard There were many changes to the readme.md. Please merge it with the new master.

**Comment by jerbia** [comment]
@cpt-redbeard Seems like you will need to rebase your pull request

**Comment by cpt-redbeard** [comment]
@jerbia I believe this was resolved with pull/106

**Pull request closed without merging**